modules/gitops-synchronizing-cluster-configs-application.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * configuring-an-openshift-cluster-with-argo-cd.adoc
+
+[id="synchronizing-cluster-configs-application_{context}"]
+= Synchronizing `cluster configs` application
+
+. In the Argo CD dashboard, notice that the `cluster-configs` Argo CD application has the statuses `Missing` and `OutOfSync`. Because the application was configured with a manual sync policy, Argo CD does not sync it automatically.
+
+. Click the *Sync* button on the `cluster-configs` tile, review the changes and then click the *Synchronize*.
+
+.  Notice that the `cluster-configs` Argo CD application now has the statuses `Healthy` and `Synced`. Click the `cluster-configs` tile to check the details of the synchronized resources and their status on the cluster.
+
+. Navigate to the OpenShift web console and click the *Application Launcher* and see a link to the _Red Hat Developer Blog_. 
```

Comment:
I don't really understand what this sentence is trying to say. If you click the Application Launcher, you will see a link?
Maybe 
"Navigate to the OpenShift web console and click the *Application Launcher* to be guided to the link to the _Red Hat Developer Blog_. "? 
Depending on what you want to say of course!

Reply:
You click the App Launcher and a link will be there (and it's usually not, we've added it in this procedure). Thinking about how to reword it. -_-